install/install.php

```diff
@@ -299,35 +311,39 @@ protected function createNotifications() {
    protected function deleteNotifications() {
       global $DB;
 
-      $this->migration->displayMessage("Delete notifications");
-
       // Define DB tables
-      $table_targets      = getTableForItemType('NotificationTarget');
-      $table_notification = getTableForItemType('Notification');
-      $table_translations = getTableForItemType('NotificationTemplateTranslation');
-      $table_templates    = getTableForItemType('NotificationTemplate');
+      $table_targets       = NotificationTarget::getTable();
+      $table_notification  = Notification::getTable();
+      $table_translations  = NotificationTemplateTranslation::getTable();
+      $table_templates     = NotificationTemplate::getTable();
 
       // Delete translations
       $query = "DELETE FROM `$table_translations`
-      WHERE `notificationtemplates_id` IN (
-      SELECT `id` FROM $table_templates WHERE `itemtype` = 'PluginFormcreatorForm_Answer')";
+                WHERE `notificationtemplates_id` IN (
```

Comment:
like below, could you also replace this direct sql by a ``deleteByCriteria`` (and so remove the getTable above)